tests/behat.yml.dist

```diff
@@ -28,6 +27,9 @@ default:
         - Drupal\nexteuropa\Context\DatabaseLogContext
         - Drupal\nexteuropa\Context\PathautoContext
         - Drupal\nexteuropa\Context\MultilingualContext
+        - Drupal\nexteuropa\Context\DrushContext:
+            drush_bin: "{{ drush.bin }}"
+            db_dump_location: '{{ drush.db.dump }}'
```

Comment:
I would add here an `enabled:` boolean property set to `false` on local environment and to `true` on Jenkins, so that we have control when dumping and restoring happens.